certbot/tests/plugins/disco_test.py

```diff
@@ -60,11 +60,9 @@ def test_entry_point_to_plugin_name_not_prefixed(self):
 
         for entry_point, name in names.items():
             self.assertEqual(
-                name, PluginEntryPoint.entry_point_to_plugin_name(entry_point, with_prefix=False))
+                name, PluginEntryPoint.entry_point_to_plugin_name(entry_point))
 
     def test_entry_point_to_plugin_name_prefixed(self):
```

Comment:
since we're no longer testing the method `entry_point_to_plugin_name`, is this test needed? perhaps rename it to indicate we're testing the `,dist.key`/`.name` fields being set correctly?

Reply:
Yes, you're right. This is just testing the mock and not any real functionality.